apps/mobile/maestro/flows/smoke-tests.yaml

```diff
@@ -0,0 +1,14 @@
+# Leather Mobile Wallet Smoke Tests
```

Comment:
This is a very basic smoke test but is effective in that it ensures the app has been built successfully and the user reaches the home screen. 

I trialled using `Gherkin` BDD style and after doing some research it seems it can be just as effective to use Gherkin comments to explain the Maestro steps